TanukiKit/User.swift

```diff
@@ -3,20 +3,67 @@ import RequestKit
 
 // MARK: model
 @objc public class User: NSObject {
-    public let id: Int
+    public var name: String?
     public var login: String?
+    public let id: Int
+    public var state: String?
+    public var avatarURL: NSURL?
+    public var webURL: NSURL?
+    public var createdAt: NSDate?
+    public var isAdmin: Bool?
     public var bio: String?
-    public var name: String?
+    public var location: String?
+    public var skype: String?
+    public var linkedin: String?
+    public var twitter: String?
+    public var websiteURL: NSURL?
+    public var lastSignInAt: NSDate?
+    public var confirmedAt: NSDate?
     public var email: String?
+    public var themeId: Int?
+    public var colorSchemeId: Int?
+    public var projectsLimit: Int?
+    public var currentSignInAt: NSDate?
+    public var canCreateGroup: Bool?
+    public var canCreateProject: Bool?
+    public var twoFactorEnabled: Bool?
+    public var external: Bool?
     public var privateToken: String?
 
     public init(_ json: [String: AnyObject]) {
         if let id = json["id"] as? Int {
-            self.id = id
+            name = json["name"] as? String
             login = json["username"] as? String
+            self.id = id
+            state = json["state"] as? String
+            if let urlString = json["avatar_url"] as? String, url = NSURL(string: urlString) {
+                avatarURL = url
+            }
+            if let urlString = json["web_url"] as? String, url = NSURL(string: urlString) {
+                webURL = url
+            }
+            createdAt = Time.rfc3339Date(json["created_at"] as? String)
+            isAdmin = json["is_admin"] as? Bool
             bio = json["bio"] as? String
-            name = json["name"] as? String
+            location = json["location"] as? String
+            skype = json["skype"] as? String
+            linkedin = json["linkedin"] as? String
+            twitter = json["twitter"] as? String
+            if let urlString = json["website_url"] as? String, url = NSURL(string: urlString) {
+                websiteURL = url
+            }
+            lastSignInAt = Time.rfc3339Date(json["last_sign_in_at"] as? String)
+            print(lastSignInAt)
```

Comment:
Can you remove the print? Don't know when that slipped in, sorry for missing it.